packages/google_sign_in/google_sign_in_web/lib/src/flexible_size_html_element_view.dart

```diff
@@ -123,10 +125,12 @@ class _FlexHtmlElementView extends State<FlexHtmlElementView> {
       size: _lastReportedSize ?? widget.initialSize ?? const Size(1, 1),
       child: HtmlElementView(
           viewType: widget.viewType,
-          onPlatformViewCreated: (int viewId) async {
-            _registerListeners(_locatePlatformViewRoot(viewId));
-            if (widget.onPlatformViewCreated != null) {
-              widget.onPlatformViewCreated!(viewId);
+          onPlatformViewCreated: (int viewId) {
+            final web.Element root =
+                ui_web.platformViewRegistry.getViewById(viewId) as web.Element;
+            _registerListeners(root);
+            if (widget.onElementCreated != null) {
```

Comment:
Optional nit: consider making a local `callback` variable, then null-testing and using that, so you don't need the force-unwrap.